_posts/2021-11-29-introducing-the-official-ugrc-api-client.md

```diff
@@ -27,7 +27,7 @@ The official [UGRC API client](https://github.com/agrc/api-client) is now availa
 ## A few highlights we are proud of
 
 - No license is needed to use the client! It is download and go.
-- The client is cross-platform. It works on [MacOS](https://github.com/agrc/api-client/releases/download/v1.6.0/UGRC.API.Client-1.6.0-x64.dmg) and [Windows](https://github.com/agrc/api-client/releases/download/v1.6.0/ugrc-api-client-1.6.0-win32-setup.exe).
+- The client is cross-platform. It works on [MacOS]({% link data/address-geocoders-locators/index.html %}#OfficialClient) and [Windows]({% link data/address-geocoders-locators/index.html %}#OfficialClient).
```

Comment:
We've still got the direct links in the paragraph above this (I can't figure out how to do a suggestion there to fix them).

Reply:
good catch.